packages/react-scripts/template/README.md

```diff
@@ -1859,11 +1847,7 @@ following into account:
 1. By default, the generated service worker file will not intercept or cache any
    cross-origin traffic, like HTTP [API requests](#integrating-with-an-api-backend),
    images, or embeds loaded from a different domain. If you would like to use a
-   runtime caching strategy for those requests, you can [`eject`](#npm-run-eject)
-   and then configure the
-   [`runtimeCaching`](https://github.com/GoogleChrome/sw-precache#runtimecaching-arrayobject)
-   option in the `SWPrecacheWebpackPlugin` section of
-   [`webpack.config.prod.js`](../config/webpack.config.prod.js).
+   runtime caching strategy for those requests, you can add in a [`runtimeCaching`](https://developers.google.com/web/tools/workbox/modules/workbox-webpack-plugin#generateSW-runtimeCaching) to section to a custom `workbox.config.js` file.
```

Comment:
The `workbox.config.js` file is something that was [alluded to](https://github.com/facebook/create-react-app/pull/4169#issuecomment-422935546), but I believe it's just hypothetical right now.

I'm assuming that that functionality is going to be added to the `next` branch at some point by the `c-r-a` maintainers, and this reference to `workbox.config.js` could at that point be linked to whatever documentation is written for that new functionality.

(If I'm getting those assumptions wrong, let me know!)